<?xml version="1.0"?>
<launch>
<!-- Тип автопилота -->
    <!-- 0 - PX4 -->
    <!-- 1 - ArduPilot -->
    <arg name="safe_zone" default="4" />
    <!-- Сторона квадрата безопастной зоны -->
    <arg name="safe_max_height" default="1" />
    <!-- Максимальная высота  безопастного полёта -->
    <arg name="safe_min_height" default="0.2" />
    <!-- Минимальная высота  безопастного полёта -->
    <arg name="tolerance" default="0.2" />
    <!-- Растояние на котором точка будет считать с достигнутой -->

    <node name="$(anon motion_server)" pkg="controll" type="motion_server.py" output="screen">
        <param name="safe_zone" value="$(arg safe_zone)" />
        <param name="safe_max_height" value="$(arg safe_max_height)" />
        <param name="safe_min_height" value="$(arg safe_min_height)" />
        <param name="tolerance" value="$(arg tolerance)" />
    </node>
</launch>